### PR TITLE
added blog.index

### PR DESCRIPTION
**Pull request by Federicofilini**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+              <a href="#" class="nav-link">Our blog</a>
+            </li>
+            <li class="nav-list__item">
+              <a href="#" class="nav-link">The Basilisks NFT</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <!-- ----------------------------------------- -->
+      <div class="ourblog-info__block">
```
Comment (andacenko) [inline blog.html:99]: It's  <main class="main main--two-col"> Please, change div.ourblog-info__block

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+        <aside class="side-bar_block">
```
Comment (andacenko) [inline blog.html:100]: Duplicated class (line 101 , line 129)
PLease, change. For example aside.aside

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+          <div class="side-bar_block">
+            <h3 class="side-bar_header">Categories</h3>
```
Comment (andacenko) [inline blog.html:102]: Use `<strong>` instead `<h3>` here

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,434 @@
...
+
+            <ul class="side-bar_list">
+              <li class="side-bar_item">
+                <a href="" class="side-bar_link">Our blog (12)</a>
```
Comment (andacenko) [inline blog.html:106]: `Our blog <span>(12)</span>` numbers should be wrapped by the span
Do this in all places below

File: blog.html [hunk 5]
```diff
@@ -0,0 +1,434 @@
...
+                <a href="" class="side-bar_link">Lauchpad (24)</a>
+              </li>
+              <li class="side-bar_item">
+                <a href="" class="side-bar_link">Crypto (430)</a>
+              </li>
+            </ul>
+          </div>
+
+          <div class="side-bar_block">
+            <h3 class="side-bar_header">Tag</h3>
```
Comment (andacenko) [inline blog.html:130]: strong instead h3

File: blog.html [hunk 6]
```diff
@@ -0,0 +1,434 @@
...
+                <p>Tony Nguyen</p>
+              </div>
+
+              <div class="box-info1">
+                <img src="" alt="" width="20" height="20" />
+                <time datetime="">20 Jun 2022</time>
+              </div>
+            </div>
+          </div> -->
+          <!-- Which option is better? ^^^^^^^^^^^^^^^^^^   -->
```
Comment (andacenko) [inline blog.html:188]: <div class="blog__info">
  <span class="blog__author">
     
     Tony Nguyen
  </span>
  <time datetime="2022-06-20" class="blog__date">
     
     20 Jun 2022
  </time>
</div>

File: blog.html [hunk 7]
```diff
@@ -0,0 +1,434 @@
...
+            Ethereum’s blockchain with real-world value. Players use
+            <a href="#">the Basilisks they have</a>
+            gathered to battle other players to win ether (ETH). In Balthazar
+            Dragons, players are immersed in a 3D open world to explore and
+            capture dragon-like beasts called Basilisks. Balthazar Dragons is an
+            upcoming fantasy role-playing game developed on the Ethereum
+            blockchain by a decentralized autonomous organization (DAO) called
+            the Balthazar DAO.
+          </p>
+          <ol>
```
Comment (andacenko) [inline blog.html:200]: Use ul instead ol there

File: blog.html [hunk 8]
```diff
@@ -0,0 +1,434 @@
...
+                explore and capture dragon-like beasts called Basilisks.
+                Balthazar Dragons is an upcoming fantasy role-playing
+                <a href="#">game developed</a> on the Ethereum blockchain by a
+                decentralized autonomous organization (DAO) called the Balthazar
+                DAO.
+              </p>
+            </li>
+          </ol>
+
+          <ol class="blog-label__container">
```
Comment (andacenko) [inline blog.html:221]: Use ul instead ol there

File: blog.html [hunk 9]
```diff
@@ -0,0 +1,434 @@
...
+            <li class="blog-label__list">
+              <img
```
Comment (andacenko) [inline blog.html:223]: Please, delete img. All decors would be in the css

File: blog.html [hunk 10]
```diff
@@ -0,0 +1,434 @@
...
+
+          <blockquote class="blog_blockquote">
+            “The first huge metaverse arena to arrange live sports, hang out and
+            even make bets and wagers”
+            <div class="blockquote-info">
+              <img src="" alt="" width="21" height="23" />
+              <p>8.5/ 10</p>
+              <p>Markout Corporation</p>
+            </div>
+          </blockquote>
```
Comment (andacenko) [inline blog.html:272]: Please, use this structure for blockquote 

```
<div class="blog__blockquote">
       <blockquote cite="https://">
             <p class="blog__blockquote--text">The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers</p>
        </blockquote>
        <p class="blog__blockquote--rating">8.5/ 10<cite>Markout Corporation</cite></p>
</div>
```

File: blog.html [hunk 11]
```diff
@@ -0,0 +1,434 @@
...
+
+          <div class="Foto_box">
+            <img src="" alt="" class="fototwo" width="448" height="390" />
+            <img src="" alt="" class="fototwo" width="448" height="390" />
```
Comment (andacenko) [inline blog.html:276]: Every img there is figure + figcaption

```
<figure>
  <img src="" w="" h="" alt="">
  <figcaption>Text</figcaption>
</figure>
```

File: blog.html [hunk 12]
```diff
@@ -0,0 +1,434 @@
...
+            their Avatars on nft.binance.com. Once you set your new unique
+            nickname, it will be standardized and used across the Binance
+            ecosystem.
+          </p>
+          <h3>NFT Avatar Generators</h3>
+          <p>
+            You can find a range of NFT marketplaces where you can buy existing
+            NFTs or list your own NFTs for sale.
+          </p>
+          <div class="blog_caption_container">
```
Comment (andacenko) [inline blog.html:303]: It's footer of article
Please use `<footer>` tag instead` <div class="blog_caption_container">`

File: blog.html [hunk 13]
```diff
@@ -0,0 +1,434 @@
...
+            <div class="caption-box">
+              <p>Tags:</p>
+              <a href="" class="captions-linc">NFT</a>
+              <a href="" class="captions-linc">ICO</a>
+              <a href="" class="captions-linc">Crypto</a>
+            </div>
+            <div class="caption-socials__box">
+              <div>
```
Comment (andacenko) [inline blog.html:311]: Unnecessary div. Please delete it.

File: blog.html [hunk 14]
```diff
@@ -0,0 +1,434 @@
...
+          <p>
+            You can find a range of NFT marketplaces where you can buy existing
+            NFTs or list your own NFTs for sale.
+          </p>
+          <div class="blog_caption_container">
+            <div class="caption-box">
+              <p>Tags:</p>
+              <a href="" class="captions-linc">NFT</a>
+              <a href="" class="captions-linc">ICO</a>
+              <a href="" class="captions-linc">Crypto</a>
```
Comment (andacenko) [inline blog.html:308]: Use ul > li > a structure here

File: blog.html [hunk 15]
```diff
@@ -0,0 +1,434 @@
...
+            </div>
+            <div class="caption-socials__box">
+              <div>
+                <p>Share:</p>
+                <a href="" class="socias"><img src="" alt="" />twitter</a>
+                <a href="" class="socias"><img src="" alt="" />facebook</a>
+                <a href="" class="socias"><img src="" alt="" />telegram</a>
```
Comment (andacenko) [inline blog.html:315]: List of social links.
Please, use ul > li > a structure here.

File: blog.html [hunk 16]
```diff
@@ -0,0 +1,434 @@
...
+                <a href="" class="side-bar_link">Lauchpad</a>
+              </li>
+              <li class="side-bar_item">
+                <a href="" class="side-bar_link">Metaverse</a>
+              </li>
+            </ul>
+          </div>
+        </aside>
+
+        <div class="ourblog-details__block">
```
Comment (andacenko) [inline blog.html:155]: Please use `<article>` tag instead div for `<div class="ourblog-details__block">`

File: blog.html [hunk 17]
```diff
@@ -0,0 +1,434 @@
...
+          <div class="blog_caption_container">
+            <div class="caption-box">
+              <p>Tags:</p>
+              <a href="" class="captions-linc">NFT</a>
+              <a href="" class="captions-linc">ICO</a>
+              <a href="" class="captions-linc">Crypto</a>
+            </div>
+            <div class="caption-socials__box">
+              <div>
+                <p>Share:</p>
```
Comment (andacenko) [inline blog.html:312]: Please `<span>` instead `<p>` here

File: blog.html [hunk 18]
```diff
@@ -0,0 +1,434 @@
...
+            ecosystem.
+          </p>
+          <h3>NFT Avatar Generators</h3>
+          <p>
+            You can find a range of NFT marketplaces where you can buy existing
+            NFTs or list your own NFTs for sale.
+          </p>
+          <div class="blog_caption_container">
+            <div class="caption-box">
+              <p>Tags:</p>
```
Comment (andacenko) [inline blog.html:305]: Please `<span>` instead `<p>` here

File: blog.html [hunk 19]
```diff
@@ -0,0 +1,434 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- ------------------------------------------------------------------------Footer -->
+    <footer class="footer">
+      <div class="footer_bar_box">
+        <div class="footer_logo_box">
+          <strong class="logo_header"
```
Comment (andacenko) [inline blog.html:326]: Please `<a>` instead `<strong>` for logo. Use should have oportunity to navigate into Home page.

File: blog.html [hunk 20]
```diff
@@ -0,0 +1,434 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blog Page</title>
+  </head>
+  <body>
+    <!-- --------------------------------------------------------------------------Header -->
+    <header class="header">
+      <h1 class="logo__text">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:11]: done

File: blog.html [hunk 21]
```diff
@@ -0,0 +1,434 @@
...
+            />
+            /> Connect
+          </button>
+        </div>
+      </div>
+    </header>
+    <!-------------------------------------------------------------------Our blog -->
+    <section class="blog_section">
+      <div class="header-nav_block">
+        <img src="" alt="Background imege" width="1728" height="133" />
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:81]: done

File: blog.html [hunk 22]
```diff
@@ -0,0 +1,434 @@
...
+
+        <h2 class="blog_header">Our blog</h2>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:83]: done

File: blog.html [hunk 23]
```diff
@@ -0,0 +1,434 @@
...
+            <li class="nav-list__item">
+              <a href="#" class="nav-link">Home</a>
+            </li>
+            <li class="nav-list__item">
+              <a href="#" class="nav-link">Our blog</a>
+            </li>
+            <li class="nav-list__item">
+              <a href="#" class="nav-link">The Basilisks NFT</a>
+            </li>
+          </ul>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:95]: Done

File: blog.html [hunk 24]
```diff
@@ -0,0 +1,434 @@
...
+
+        <div class="ourblog-details__block">
+          <img
+            src=""
+            alt=" Big Dragon"
+            class="blog-main_img"
+            width="920"
+            height="557"
+          />
+
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:163]: Done

File: blog.html (hunk 1 above)
Comment (Federicofilini) [reply blog.html:99]: Done

File: blog.html (hunk 4 above)
Comment (Federicofilini) [reply blog.html:106]: done

File: blog.html (hunk 5 above)
Comment (Federicofilini) [reply blog.html:130]: done

File: blog.html (hunk 6 above)
Comment (Federicofilini) [reply blog.html:188]: Done

File: blog.html (hunk 8 above)
Comment (Federicofilini) [reply blog.html:221]: done

File: blog.html (hunk 7 above)
Comment (Federicofilini) [reply blog.html:200]: Done

File: blog.html (hunk 9 above)
Comment (Federicofilini) [reply blog.html:223]: Done

File: blog.html (hunk 3 above)
Comment (Federicofilini) [reply blog.html:102]: done

File: blog.html (hunk 10 above)
Comment (Federicofilini) [reply blog.html:272]: done

File: blog.html (hunk 11 above)
Comment (Federicofilini) [reply blog.html:276]: Done

File: blog.html (hunk 12 above)
Comment (Federicofilini) [reply blog.html:303]: done

File: blog.html (hunk 13 above)
Comment (Federicofilini) [reply blog.html:311]: Done

File: blog.html (hunk 14 above)
Comment (Federicofilini) [reply blog.html:308]: Done

File: blog.html (hunk 16 above)
Comment (Federicofilini) [reply blog.html:155]: done

File: blog.html (hunk 17 above)
Comment (Federicofilini) [reply blog.html:312]: done

File: blog.html (hunk 18 above)
Comment (Federicofilini) [reply blog.html:305]: done

File: blog.html (hunk 19 above)
Comment (Federicofilini) [reply blog.html:326]: done

File: blog.html [hunk 25]
```diff
@@ -0,0 +1,405 @@
...
+      <div class="header-nav_block">
+        <h1 class="blog_header">Our blog</h1>
+        <ul class="blog-page_breadcrumbs">
+          <li><a class="home-page_link" href="">Home</a></li>
+          <li><a class="our-blog-page_link" href="">Our blog</a></li>
+          <li><p>The Basilisks NFT</p></li>
+        </ul>
+      </div>
+      <!-- ----------------------------------------- -->
+      <div class="ourblog-info__block">
```
Comment (andacenko) [inline blog.html:89]: It's `<main class="main main--two-col"> `Please, change div.ourblog-info__block to main

File: blog.html [hunk 26]
```diff
@@ -0,0 +1,405 @@
...
+            their Avatars on nft.binance.com. Once you set your new unique
+            nickname, it will be standardized and used across the Binance
+            ecosystem.
+          </p>
+          <h3>NFT Avatar Generators</h3>
+          <p>
+            You can find a range of NFT marketplaces where you can buy existing
+            NFTs or list your own NFTs for sale.
+          </p>
+          <div class="blog_tags_container">
```
Comment (andacenko) [inline blog.html:274]: `<div class="blog_tags_container"> `it's footer of article
Please use `<footer class="...">` tag instead`

File: blog.html [hunk 27]
```diff
@@ -0,0 +1,405 @@
...
+                <li class="tag_item">
+                  <a href="" class="tag-linc">ICO</a>
+                </li>
+                <li class="tag_item">
+                  <a href="" class="tag-linc">Crypto</a>
+                </li>
+              </ul>
+            </div>
+            <div class="socials__box">
+              <div>
```
Comment (andacenko) [inline blog.html:290]: Please, delete div without class
line 290

File: blog.html [hunk 28]
```diff
@@ -0,0 +1,405 @@
...
+        <ul class="blog-page_breadcrumbs">
+          <li><a class="home-page_link" href="">Home</a></li>
+          <li><a class="our-blog-page_link" href="">Our blog</a></li>
+          <li><p>The Basilisks NFT</p></li>
+        </ul>
+      </div>
+      <!-- ----------------------------------------- -->
+      <div class="ourblog-info__block">
+        <aside class="aside">
+          <div class="side-bar_block-one">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:91]: Done

File: blog.html (hunk 26 above)
Comment (Federicofilini) [reply blog.html:274]: Done

File: blog.html [hunk 29]
```diff
@@ -0,0 +1,405 @@
...
+              <li class="side-bar_item">
+                <a href="" class="side-bar_link">Lauchpad <span>(24)</span></a>
+              </li>
+              <li class="side-bar_item">
+                <a href="" class="side-bar_link">Crypto <span>(430)</span></a>
+              </li>
+            </ul>
+          </div>
+
+          <div class="side-bar_block block-two">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:119]: Done

File: blog.html (hunk 25 above)
Comment (Federicofilini) [reply blog.html:89]: Done

File: blog.html (hunk 27 above)
Comment (Federicofilini) [reply blog.html:290]: done

**Review by andacenko — APPROVED** [review]

**Pull request merged**